sway-lib-core/src/ops.sw

```diff
@@ -1258,3 +1292,19 @@ pub fn ok_str_eq() {
     assert("" != "a");
     assert("a" != "b");
 }
+
+fn flags() -> u64 {
```

Comment:
Is this function needed?

Reply:
I suppose not at the very moment, but in the future if you want to add a panic_on_unsafe_math_enabled() function you would need it seperate